_posts/2023-06-28-OLS-at-CollabW23.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "The OLS community at CollabW23"
+authors:
+- pherterich
+- gemmaturon
+- saranjeetkaur
+- amangoel185
+image: /images/2023-06-28-OLS-at-CW23.jpg
+---
+
+From 2 to 4 May 2023, our sponsors and friends at the Software Sustainability Institute hosted their annual flagship event, [Collaborations Workshop](https://software.ac.uk/cw23). 
+CollabW23 is a special event and truly collaborative with many interactive sessions. 
+This year, it was run as a hybrid format and it was great to see the effort of bringing people together in person and remotely.  
+
+For me, it was a great opportunity to meet many of the OLS community in person for the first time. 
+Together with Resident Fellow Mayya Sundukova I also hosted a session on self care and how OLS provides exercises to help assess and plan for more self-care.
+
+## Personal highlights from CollabW23
+
+This year’s theme was “Sustainable Career Development for those in the research software community: looking after your software, your career, and yourself” which is a topic that resonated very much with me as I spent last year thinking much about burnout, self care and career changes that might follow from that. 
+The [keynote panel](https://www.youtube.com/watch?v=vqVtB3B8vVo) on careers was incredibly insightful and I loved how vulnerable and honest all speakers were.
+
+During the discussion session, my group discussed how to build collaborative teams that have different experience levels, locations, working patterns and contract types. 
+OLS is a distributed team with many people working part time, so how we work well together is an important conversation to have (speed blog summarising the discussion to come!).
+
+Together with Mayya, I facilitated a session on self care (slides can be found on [Zenodo](https://doi.org/10.5281/zenodo.8060439)). 
+I was the in-person facilitator for our session while Mayya led the discussion with our online attendees. 
+As we had technical issues during our session, this was a great way of co- facilitating as Mayya could make sure remote participants are looked after while I kept a conversation in the room going until our technical issues were resolved.
+
+SSI/OLS Fellow Reina Camacho Toro led a session exploring if current concepts and practices around Open Science are global and universal. 
+She and her co-facilitators piloted a survey aiming to understand the current barriers, successful practices and possible improvements needed for the implementation of Open Science in Latin America.
+
+I also attended a session on (agile) project management and a workshop playing through an evil collaboration scenario. 
+It is great to see these topics being so central in a conference as they are usually seen as low priority and getting in the way of getting work done. 
+
+## OLS community activities and highlights
+
```

Comment:
It might be good to have an introduction sentence to mention that now different people will share their experience. It is not really clear for me from the title

Reply:
Good point - added something now!